apps/trambyt/README.md

```diff
@@ -0,0 +1,8 @@
+# Trambyt
+
+Shows upcoming departures from Västtrafik, western Sweden public transport. The use can configure
+a transit stop and the app will show the next departures.
+
+![Trambyt screengrab](trambyt.png)
+
+Dedicated to matslina for his deep love of fish.
```

Comment:
I hope you're running [Fishbyt](https://github.com/tidbyt/community/blob/main/apps/fishbyt/fishbyt.star) on your Tidbyt!